Source/Particles/RigidInjectedParticleContainer.cpp

```diff
@@ -252,9 +252,6 @@ RigidInjectedParticleContainer::PushPX (WarpXParIter& pti,
         amrex::ParticleReal* AMREX_RESTRICT x_save = xp_save.dataPtr();
         amrex::ParticleReal* AMREX_RESTRICT y_save = yp_save.dataPtr();
         amrex::ParticleReal* AMREX_RESTRICT z_save = zp_save.dataPtr();
```

Comment:
Saving and re-setting `u` to its original value is not needed anymore since the scaling factor for E and B is 0 for particles to the left of the plane.